test/cdp-proxy/reactNativeCDPProxy.test.ts

```diff
@@ -0,0 +1,318 @@
+
+      const deliverlyTest = function (messageHandler: ICDPMessageHandler) {
+
+        return test(`Messages should be delivered correctly with ${(<Object>messageHandler).constructor.name}`, async () => {
+          const targetMessageStart = {method: "Target.start", params: {reason: "test"}};
+          const debuggerMessageStart = {method: "Debugger.start", params: {reason: "test"}};
+
+          const messageFromTarget = await new Promise((resolve) => {
+            targetConnection?.send(targetMessageStart);
+
+            debugConnection?.onCommand((evt: IProtocolCommand) => {
+              resolve(evt);
+            });
+          })
+           .then((evt) => {
+            return evt;
+          });
+
+          const messageFromDebugger = await new Promise((resolve) => {
+            debugConnection?.send(debuggerMessageStart);
+
+            targetConnection?.onCommand((evt: IProtocolCommand) => {
+              resolve(evt);
+            });
+          })
+           .then((evt) => {
+            return evt;
+          });
+
+          assert.deepEqual(messageFromTarget, targetMessageStart);
+          assert.deepEqual(messageFromDebugger, debuggerMessageStart);
+        });
+      };
+
+      suite(`${(<Object>rnHandler).constructor.name}`, () => {
+
+        suiteSetup(async () => {
+          rnHandler = new RnCDPMessageHandler();
+          Object.assign(proxy, {CDPMessageHandler: rnHandler});
+        });
+
+        deliverlyTest(rnHandler);
+
+        test(`Message from target with method '${CDP_API_NAMES.DEBUGGER_PAUSED}' in first time should transorm reason to 'Break on start'`, async () => {
+          const targetMessagePaused = {method: CDP_API_NAMES.DEBUGGER_PAUSED, params: {reason: "other"}};
+
+          const messageFromTarget = await new Promise((resolve) => {
+            targetConnection?.send(targetMessagePaused);
+
+            debugConnection?.onCommand((evt: IProtocolCommand) => {
+              resolve(evt);
+            });
+          })
+           .then((evt) => {
+             targetMessagePaused.params.reason = "Break on start";
+             return evt;
+          });
+
+          assert.deepEqual(messageFromTarget, targetMessagePaused);
+        });
+
+      });
+
+      suite(`${(<Object>directHandler).constructor.name}`, () => {
+        const HERMES_NATIVE_FUNCTION_NAME: string = "(native)";
+        const HERMES_NATIVE_FUNCTION_SCRIPT_ID: string = "4294967295";
+        const ARRAY_REQUEST_PHRASE_MARKER: string = "Object.getOwnPropertyDescriptor";
+
+        suiteSetup(async () => {
+          directHandler = new DirectCDPMessageHandler();
+          Object.assign(proxy, {CDPMessageHandler: directHandler});
+        });
+
+        deliverlyTest(directHandler);
+
+        test(`Message from target with method '${CDP_API_NAMES.DEBUGGER_PAUSED}' should filter callFrames with Hermes function name and script id`, async () => {
+          const callFrames: any = [
```

Comment:
I think it could be better to store such objects instances in a JSON files, it could make the code more readable and the objects will be more easier to understand. Thoughts?